tests/evaluate/test_default_evaluator.py

```diff
@@ -2048,8 +2096,56 @@ def test_evaluate_question_answering_without_targets():
     artifacts = [a.path for a in client.list_artifacts(run.info.run_id)]
     assert "eval_results_table.json" in artifacts
     logged_data = pd.DataFrame(**results.artifacts["eval_results_table"].content)
-    pd.testing.assert_frame_equal(logged_data, data.assign(outputs=["a", "b"]))
-    assert results.metrics == {}
+    validate_question_answering_logged_data(logged_data, False)
+    assert set(results.metrics.keys()) == {
+        "mean_perplexity",
+        "toxicity_ratio",
+        "mean_ari_grade_level",
+        "mean_flesch_kincaid_grade_level",
+    }
+    assert results.metrics["toxicity_ratio"] == 0.0
+
+
+def validate_text_summarization_logged_data(logged_data, with_targets=True):
+    if with_targets:
+        columns = [
+            "text",
+            "summary",
+            "outputs",
+            "rouge1",
+            "rouge2",
+            "rougeL",
+            "rougeLsum",
+            "toxicity",
+            "flesch_kincaid_grade_level",
+            "ari_grade_level",
+            "perplexity",
+        ]
+    else:
+        columns = [
+            "text",
+            "outputs",
+            "toxicity",
+            "flesch_kincaid_grade_level",
+            "ari_grade_level",
+            "perplexity",
+        ]
```

Comment:
https://github.com/mlflow/mlflow/pull/9049/files#r1273043680 we can make the same change here.